docker/cmsweb-base/Dockerfile

```diff
@@ -7,7 +7,7 @@ ADD slc7-cernonly.repo /etc/yum.repos.d/slc7-cernonly.repo
 # see https://developers.redhat.com/blog/2016/03/09/more-about-docker-images-size/
 RUN yum install -y sudo cern-get-certificate fetch-crl \
     CERN-CA-certs ca-certificates dummy-ca-certs ca-policy-lcg ca-policy-egi-core \
-    ca_EG-GRID ca_CERN-GridCA ca_CERN-LCG-IOTA-CA ca_CERN-Root-2 \
+    ca_EG-GRID ca_CERN-GridCA ca_CERN-LCG-IOTA-CA ca_CERN-Root-2 wlcg-voms-cms \
```

Comment:
Hi @todor-ivanov, can you please change this to:
`RUN yum install -y --nogpgcheck wlcg-voms-cms` as a separate command?

The initial docker image gives the following error: `GPG key retrieval failed: [Errno 14] curl#37 - "Couldn't open file /etc/pki/rpm-gpg/RPM-GPG-KEY-wlcg"`

But after the suggested changes, it builds seamlessly, and has the relevant files:
```
[root@be933250ba90 vomsdir]# cd /etc/grid-security/vomsdir
[root@be933250ba90 vomsdir]# ls
cms
[root@be933250ba90 vomsdir]# cd /etc/vomses
[root@be933250ba90 vomses]# ls
cms-lcg-voms2.cern.ch  cms-voms2.cern.ch
[root@be933250ba90 vomses]# exit
```

Reply:
Thanks  for testing this @arooshap 
I just moved  that line  as a separate run command as you suggest. 
Could you please take another look?